src/test/scala-2.11/markets/participants/strategies/TestMarketOrderTradingStrategy.scala

```diff
@@ -0,0 +1,34 @@
+package markets.participants.strategies
+
+import akka.agent.Agent
+
+import markets.tickers.Tick
+import markets.tradables.Tradable
+
+import scala.collection.mutable
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [It's preferred to use immutable collections instead of mutable ones](https://www.codacy.com/app/davidrpugh/markets-sandbox/file/1704920101/issues/source?bid=2921361&fileBranchId=3046267#l8)